### PR TITLE
Add command to identify and merge some paths #3607

### PR DESCRIPTION
**Pull request by Chatewgne**
#3607

### CODE REVIEW

File: geotrek/core/tests/test_commands.py [hunk 1]
```diff
@@ -626,3 +626,61 @@ def test_split_reorder_fail(self):
...
+        with self.assertRaises(Path.DoesNotExist):
+            Path.objects.get(pk=self.p7.pk)
+        call_command('merge_segmented_paths', stdout=output)
+        #  After second call :
+        #    p1      p2      p3               p8                 p9
+        # +-------+------+-------+----------------------------+------+
+        #                |
+        #                |  p4
+        #                |
+        #                +
```
Comment (babastienne) [inline geotrek/core/tests/test_commands.py:683]: Can't we achieve the following result with the this command ?

```python
#  After x call :
#        p1                         p3              
# +--------------+-------------------------------------------+
#                |
#                |  p4
#                |
#                +
```

Comment (babastienne) [reply geotrek/core/tests/test_commands.py:683]: It works in the web interface of Geotrek-Admin, that's why I was asking. Isn't the same function called behind ?

**Comment by babastienne** [comment]
:pray: :heart:

**Review by babastienne — CHANGES_REQUESTED** [review]
Can you add some explanations in the[ "exploitation command" section of the documentation](https://geotrek.readthedocs.io/en/2.99.0/install/import.html#exploitation-commands) please ?

**Comment by babastienne** [comment]
> > Can you add some explanations in the[ "exploitation command" section of the documentation](https://geotrek.readthedocs.io/en/2.99.0/install/import.html#exploitation-commands) please ?
> 
> image

Great, thanks @Chatewgne. Could you put the warning in a "danger" section in order for it to but more visible ? Like this : 

image

**Comment by Chatewgne** [comment]
> Great, thanks @Chatewgne. Could you put the warning in a "danger" section in order for it to but more visible ? Like this 
image

**Review by babastienne — APPROVED** [review]
Looks good to me.

Already used on multiple instances and it works fine !

**Comment by babastienne** [comment]
I used this command already and recently again on two instances without identifying any issue. Could this PR be merged soon ?

**Comment by Chatewgne** [comment]
> I used this command already and recently again on two instances without identifying any issue. Could this PR be merged soon or is it missing something ?

Glad to hear it worked fine, I'll rebase and merge it :hugs:

**Pull request merged**